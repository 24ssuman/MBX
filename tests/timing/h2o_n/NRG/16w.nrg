SYSTEM h2o
MOLECULE
MONOMER h2o
O  0.025016  -0.22529  -0.78734
H  0.18573  -1.1486  -0.68604
H  0.88884  0.22847  -0.79632
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -1.9846  -0.017433  0.86606
H  -2.3656  0.83031  1.0299
H  -1.1846  0.19525  0.25791
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -0.14754  -1.036  2.4294
H  -0.16774  -2.025  2.4324
H  -0.8704  -0.59255  2.0189
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  2.4639  -0.030955  1.0414
H  1.6496  -0.4718  1.2655
H  2.9399  0.006554  1.857
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  2.1591  1.3705  -1.0761
H  2.5991  1.203  -1.8935
H  2.5096  0.95004  -0.30459
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  0.02805  -3.0207  -0.52713
H  -0.15705  -2.9941  0.45757
H  -0.83422  -3.1244  -0.94425
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -0.26348  3.1069  -0.7793
H  0.588  2.5735  -0.79602
H  -0.44978  3.3069  0.12581
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -1.2585  -0.16771  -3.2208
H  -0.39567  -0.6098  -3.3878
H  -1.2931  0.01995  -2.2203
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -2.3492  1.9601  -1.8906
H  -2.3918  2.0144  -2.828
H  -1.4647  2.2943  -1.6157
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -2.2285  -2.3753  -1.4532
H  -2.1394  -1.7775  -2.1818
H  -2.7642  -3.161  -1.8199
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -0.70486  3.4958  1.9828
H  -0.46944  4.098  2.7246
H  -1.4741  3.0103  2.2946
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -3.1104  -2.1271  2.0618
H  -3.4379  -2.8617  1.4922
H  -2.696  -1.4769  1.5435
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  1.0838  1.3423  3.8933
H  0.75549  0.63605  3.3046
H  1.3731  1.9932  3.2496
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  3.9989  -1.4052  -0.51823
H  3.3388  -0.97157  -0.059856
H  3.6231  -1.9586  -1.2413
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -3.3163  1.8319  2.3404
H  -3.9762  2.5071  2.6495
H  -3.5909  1.0589  2.824
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -0.28456  -3.7131  2.3739
H  0.53801  -4.1758  2.5047
H  -0.94809  -4.119  2.8712
ENDMON
ENDMOL
ENDSYS
