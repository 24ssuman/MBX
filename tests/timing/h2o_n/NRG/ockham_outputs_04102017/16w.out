Energies without gradients:
system[....0]=     1.1339327668e+02    kcal/mol
Energies with gradients:
system[....0]=     1.1339327668e+02    kcal/mol
Energy:(nograd) 17 milliseconds
Energy:(grad) 32 milliseconds
1B:(nograd) 0 milliseconds
1B:(grad) 0 milliseconds
2B:(nograd) 2 milliseconds
2B:(grad) 4 milliseconds
3B:(nograd) 11 milliseconds
3B:(grad) 22 milliseconds
Get and set 2b clusters: 0 milliseconds
Get and set 3b clusters: 0 milliseconds
